### ISSUE TITLE
Upgrade to iPython 3.2.2 

### ISSUE THREAD

**Issue by jakirkham** [ContinuumIO/anaconda-issues#457]
A series of bug fixes have been released to the iPython 3.x line as release 3.2.2 ( https://github.com/ipython/ipython/releases/tag/rel-3.2.2 ). It would be nice if this bug fix release was made available to users.

**Comment by jakirkham** [comment]
Any traction on this? It has been nearly a month since that release came out.

**Comment by msarahan** [comment]
Hi,  Thanks for the ping. I will ensure that these make it into the pending
Anaconda 2.4 release. They will be available to conda install soon (this
weekend).

**Comment by jakirkham** [comment]
Fantastic! Thanks @msarahan.

**Comment by msarahan** [comment]
@jakirkham this is a little bit complicated.  We have updated the Anaconda version of IPython to the 4.x release.  We do not officially support IPython 3.x in Anaconda anymore.  I am happy to update the recipe on conda-recipes for you (this is very simple, and amounts to just changing the git_tag entry).  I can't provide updated packages for IPython 3 in the official repo's, though.  Would you be OK with using an anaconda.org channel?

**Comment by jakirkham** [comment]
It would be nice if this weren't the case considering how recent and significant the departure between these two versions is. That being said, if your hands are tied, I understand.

Updating the conda-recipes would be nice, I think there were a few things missing last time I tried. Apologies for not documenting this in an issue.

If it's in another channel that is fine.

Part of the reason I raise this is I found an incompatibility between iPython and pymongo due to a breaking change in pymongo3. This is fixed in iPython 3.2.2 and will be in the next release of ipyparallel (in `master`).
